From 3ad07afb89deda179c668bb726d6a17cb73efdc6 Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Tue, 22 Oct 2024 15:42:27 +0100
Subject: [PATCH 19/41] vlc_vout_window: Add a scaling fraction from VLC to
 native

Add members to vout_window_t to hold the scale factor between
reported sizes and coords and window system native coords.
This allows a module to create extra planes of the correct
size and position in wayland.


Upstream-Status: Inappropriate

RPI-Distro repo forks original vlc and applies patches

to enable raspiberry pi support.
---

 include/vlc_vout_window.h | 3 +++
 1 file changed, 3 insertions(+)

--- a/include/vlc_vout_window.h
+++ b/include/vlc_vout_window.h
@@ -205,6 +205,9 @@ struct vout_window_t {
      */
      vlc_mutex_t handle_lock;
      unsigned int handle_seq;
+     /* Scale factor from reported size to physical, 0/0 if unknown */
+     int scale_num;
+     int scale_den;
 };
 
 /**
